ts/features/uaDonations/navigation/navigator.ts

```diff
@@ -0,0 +1,20 @@
+import { createStackNavigator } from "react-navigation-stack";
+import { UAWebViewScreen } from "../screens/UAWebViewScreen";
+import UADONATION_ROUTES from "./routes";
+
+const UADonationNavigator = createStackNavigator(
+  {
+    [UADONATION_ROUTES.WEBVIEW]: {
+      screen: UAWebViewScreen
+    }
+  },
+  {
+    // Let each screen handle the header and navigation
+    headerMode: "none",
+    defaultNavigationOptions: {
+      gesturesEnabled: false
+    }
+  }
+);
```

Comment:
In order to be ready for the upcoming update with `react-navigation v5`, if no other screens are expected in this navigation flow, could we avoid to add a stack navigator and add direcly the 
```
    [UADONATION_ROUTES.WEBVIEW]: {
      screen: UAWebViewScreen
    }
```

in the `AppNavigator`?
If so, could you add also:

`[UADONATION_ROUTES.WEBVIEW] : undefined;`

in https://github.com/pagopa/io-app/blob/master/ts/navigation/params/AppParamsList.ts#L14-L13 ?

Reply:
sure! 
I hope I hit the point
https://github.com/pagopa/io-app/pull/3813/commits/c877e0f81968a2dd35220328a214fe3ba9d3c2d9

Reply:
This for the paramslist, but if no other screens are expected, we should also remove the UADonationNavigator and use direcly:

```
  [UADONATION_ROUTES.WEBVIEW]: {
      screen: UAWebViewScreen
    }
```

in `AppNavigator`.

Reply:
Thank for your patience
addressed here https://github.com/pagopa/io-app/pull/3813/commits/93fd9c7875758b1768acf468c59eb76d696d0fc0